de_workloads/data_processing/silver/data_factory/adf_pipelines.tf

```diff
@@ -0,0 +1,28 @@
+resource "random_uuid" "deployment" {
+}
+
+resource "azurerm_resource_group_template_deployment" "pipeline_silver" {
+#   count               = var.include_data_quality == false ? 1 : 0
+  name                = "silver-${random_uuid.deployment.result}"
+  resource_group_name = var.data_factory_resource_group_name
+  deployment_mode     = "Incremental"
```

Comment:
would this better to be set a variable, rather than a hardcoded value, its almost certain that we want this to incremental but its always best practice to have variable and set the default

Reply:
done